go.mod

```diff
@@ -23,4 +23,5 @@ require (
 	github.com/urfave/cli v1.22.1
 	go4.org v0.0.0-20190218023631-ce4c26f7be8e // indirect
 	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/client-go v0.18.0 // indirect
```

Comment:
Is this still necessary?

Reply:
Good catch! Repushed. We need to change `make vendor` to run `go mod tidy` as a last step. It's currently not cleaning up and I assume because we run it before `go mod vendor`.

Reply:
https://github.com/containers/skopeo/pull/888